<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiamondSheetsClass</class>
 <widget class="QMainWindow" name="DiamondSheetsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1034</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DiamondSheets</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="maximumSize">
    <size>
     <width>280</width>
     <height>210</height>
    </size>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <widget class="QTableView" name="tableView">
      <property name="showGrid">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1034</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newFile">
   <property name="text">
    <string>Новий</string>
   </property>
   <property name="toolTip">
    <string>Новий файл</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="openFile">
   <property name="text">
    <string>Відкрити</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="saveFile">
   <property name="text">
    <string>Зберегти</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="showInfo">
   <property name="text">
    <string>Про програму</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Авторизуватись</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>Тест2</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="DiamondSheets.qrc"/>
 </resources>
 <connections/>
</ui>
